tests/unit/utils_test.py

```diff
@@ -404,18 +404,26 @@ def test_parse_host(self):
             'https://kokia.jp:2375': 'https://kokia.jp:2375',
             'unix:///var/run/docker.sock': 'http+unix:///var/run/docker.sock',
             'unix://': 'http+unix://var/run/docker.sock',
+            '12.234.45.127:2375/docker/engine': (
+                'http://12.234.45.127:2375/docker/engine'
+            ),
             'somehost.net:80/service/swarm': (
                 'http://somehost.net:80/service/swarm'
             ),
             'npipe:////./pipe/docker_engine': 'npipe:////./pipe/docker_engine',
+            '[fd12::82d1]:2375': 'http://[fd12::82d1]:2375',
+            'https://[fd12:5672::12aa]:1090': 'https://[fd12:5672::12aa]:1090',
+            '[fd12::82d1]:2375/docker/engine': (
+                'http://[fd12::82d1]:2375/docker/engine'
+            ),
```

Comment:
What do you mean by normal? Both are valid:
http://v6decode.com/#address=fd12%3A%3A82d1
http://v6decode.com/#address=fd12%3A5672%3A%3A12aa

The brackets are conform to [RFC 2732](https://www.ietf.org/rfc/rfc2732.txt)